doc/references.bib

```diff
@@ -0,0 +1,166 @@
+% Encoding: UTF-8
```

Comment:
Why are these particular papers here and where are they being cited?

Reply:
These are included in the structure learning algorithms. I can remove them for now and re-add them later tho.